lib/rubocop/cop/rspec/eq.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Use `eq` instead of `be ==` to compare objects.
+      #
+      # @example
+      #   # bad
+      #   expect(foo).to be == true
+      #   expect(bar).not_to be === 1
```

Comment:
=== is different from eq/eql/equal/==. I regret bringing it up

Reply:
Ah that’s right 😅 Thank you. I updated this PR.